setup.py

```diff
@@ -29,31 +30,33 @@
 Alternatively, you can install from source with
 
     ./setup.py install --user
-"""
 
+"""
 
-from __future__ import print_function
 
 import sys
+
+from io import open
```

Comment:
`from io import open` gets Python3-enabled `open` on Python2, and does nothing on Python3.  At least allows users running setup.py from Python2 to get into the warning and setup() call.